data/templates/setup_iso.py

```diff
@@ -0,0 +1,150 @@
+import argparse
+import subprocess
+import os
+import errno
+import os.path
+import tempfile
+import atexit
+import sys
+import shutil
+import re
+import urllib
+
```

Comment:
Or at least a `-h` flag handler, if you still intend for this to be called manually (as you say below).